CONTRIBUTING.md

```diff
@@ -1,19 +1,39 @@
 # Contributing
 
 ## Dev Container
-The easiest way to start contributing is via our Dev Container. To open the project in vscode you will need the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers). For codespaces you will need to [create a new codespace](https://codespace.new/Mooncake-Labs/pg_mooncake).
 
-With the extension installed you can run the following from the `Command Palette` to get started
-```
-> Dev Containers: Clone Repository in Container Volume...
-```
+The easiest way to start contributing is by using our **Dev Container**.
+You can choose between two approaches:
 
-In the subsequent popup paste the url to the repo and hit enter.
-```
-https://github.com/Mooncake-Labs/moonlink
-```
+---
+
+### 1. Local VS Code (Recommend)
+
+**Prerequisites:**
+- [Docker](https://docs.docker.com/get-docker/) (or Podman)
+- [VS Code](https://code.visualstudio.com/)
+- [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+**Steps:**
+
+1. Clone the repository
+    ```bash
+    git clone https://github.com/Mooncake-Labs/moonlink.git
+    cd moonlink
+    ```
+2. Open in VS Code
+3. Inside VS Code, run the following from the `Command Palette` to get started
+    ```
+    > Dev Containers: Reopen Folder in Container
```

Comment:
curious is it `reopen` or `open` on your end?
<img width="598" height="227" alt="image" src="https://github.com/user-attachments/assets/db2a8af0-2b1c-4255-bdd7-a61c16f783d7" />